src/features/selectedPost/selectedPost.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Add a brief description of the module at the beginning of the file as a comment.